sdk/communication/azure-communication-rooms/azure/communication/rooms/_rooms_client.py

```diff
@@ -98,27 +94,27 @@ def from_connection_string(cls, conn_str,  # type: str
     @distributed_trace
     def create_room(
         self,
-        valid_from=None, # type: Optional[datetime]
-        valid_until=None, # type: Optional[datetime]
-        room_join_policy=None, # type: Optional[RoomJoinPolicy]
-        participants=None, # type: Optional[List[RoomParticipant]]
+        *,
+        valid_from: Optional[datetime] = None,
+        valid_until: Optional[datetime] = None,
+        room_join_policy: Optional[RoomJoinPolicy] = None,
+        participants: Optional[list[RoomParticipant]]=None,
         **kwargs
-    ):
-        # type: (...) -> RoomModel
+    ) -> CommunicationRoom:
         """Create a new room.
 
-        :param valid_from: The timestamp from when the room is open for joining. The timestamp is in
+        :param valid_from: (Optional)The timestamp from when the room is open for joining. The timestamp is in
          RFC3339 format: ``yyyy-MM-ddTHH:mm:ssZ``.
-        :type valid_from: ~datetime
-        :param valid_until: The timestamp from when the room can no longer be joined. The timestamp
+        :type valid_from: (Optional)datetime
+        :param valid_until: (Optional)The timestamp from when the room can no longer be joined. The timestamp
          is in RFC3339 format: ``yyyy-MM-ddTHH:mm:ssZ``.
-        :type valid_until: ~datetime
-        :keyword room_join_policy: (Optional)The join policy of the room.
-        :paramtype room_join_policy: (Optional)RoomJoinPolicy
-        :keyword participants: (Optional) Collection of identities invited to the room.
-        :paramtype participants: (Optional)list[RoomParticipant]
+        :type valid_until: (Optional)datetime
+        :param room_join_policy: (Optional)The join policy of the room.
+        :type room_join_policy: (Optional)RoomJoinPolicy
+        :param participants: (Optional) Collection of identities invited to the room.
+        :type participants: (Optional)list[RoomParticipant]
```

Comment:
using docstring as what the doc https://review.docs.microsoft.com/en-us/help/onboard/admin/reference/python/documenting-api?branch=main#methods-and-functions suggests